tests/test_unit.py

```diff
@@ -37,6 +38,7 @@ def test_embed_english_sentence():
     MODEL = "text-embedding-ada-002"
 
     embedder_block_text = OpenAIEmbedderPlugin(config={
+        "api_key":"",
```

Comment:
Thinking out loud -- unsure if this is excessive testing, but given the room for error with `"" v None` as settings, might be good to have the tests iterate across both

Reply:
The plugin can never get None as a config setting from the engine. None config values are explicitly disallowed.